components/style/servo/selector_parser.rs

```diff
@@ -430,11 +430,11 @@ impl<'a> ::selectors::Parser for SelectorParser<'a> {
     }
 
     fn default_namespace(&self) -> Option<Namespace> {
-        self.namespaces.default.clone()
+        self.namespaces.default.clone().map(|(ns, _)| ns)
```

Comment:
This should optimize out? At least, it will optimize as much as `.as_ref().map().clone()` will